configure

```diff
@@ -19,12 +18,6 @@ from pathlib import Path
 amrex_current_version = "25.03"
 # Required minimum mpich version - ignore unless using ASan
 mpich_minimum_version = "0.0.0"
-# Python versions
-python_minimum_major = 3
-python_minimum_minor = 5
-
-
-python_version = str(sys.version_info.major) + "." + str(sys.version_info.minor)
```

Comment:
Hmm. I'm hesitant to remove this - I don't like the idea of scattering version requirements around unless necessary. I'm open to a more systematic solution and realize the need to enforce consistency. But I really want to keep all the version information together.